pkg/config/config.go

```diff
@@ -160,6 +166,45 @@ func LoadFile(filename string, expandEnvVars bool, c *Config) error {
 	return LoadBytes(buf, expandEnvVars, c)
 }
 
+// LoadRemote reads a config from url
+func LoadRemote(url string, expandEnvVars bool, c *Config) error {
```

Comment:
nit: I'd rather `rc.Retrieve` return `([]byte, error)` so we don't have to propagate expandEnvVars (and so it could exist in its own package)

Reply:
This seems reasonable; I almost went this route initially.